### ISSUE TITLE
Unresolvable Spring properties

### ISSUE THREAD

**Issue by piotrooo** [zalando/logbook#1574]
## Detailed Description
IntelliJ reports an unresolvable property.

## Context
Currently IntelliJ cannot suggest available properties. They works, but IntelliJ shows errors:

![Screenshot from 2023-07-10 10-41-20](https://github.com/zalando/logbook/assets/2005054/c56a2ceb-2702-4fbc-88ae-8825c9152a92)

Auto-completion of the properties provided by a Spring module, could be a really good developer experience.

## Possible Implementation
Metadata `spring-configuration-metadata.json` should be generated instead of `additional-spring-configuration-metadata.json`.

## Your Environment
* Version used: 3.1.0

**Comment by kasmarian** [comment]
Hi @piotrooo 

The property you want to use is `logbook.minimum-status`, not `logbook.status-at-least`, see [relevant section in readme](https://github.com/zalando/logbook/#configuration) and the [LogbookAutoConfiguration](https://github.com/zalando/logbook/blob/d451bb409c087e8d737267e4175c68c4c25fb004/logbook-spring-boot-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookAutoConfiguration.java#L232) class.

**Comment by piotrooo** [comment]
Thanks for tip, that was missclick. But that in fact is a reason of this feature/fix. If my IDE suggest me correct values (based on e.g. `LogbookProperties`) that kind of errors will disappear. It's gives a better developer experience.

**Comment by kasmarian** [comment]
For this property I'm getting the hint from IntelliJ
![image](https://github.com/zalando/logbook/assets/11900064/99a450f7-f711-484b-af05-5e8a3dc81470)

I'm using IntelliJ IDEA 2023.1.3 (Ultimate Edition) Build #IU-231.9161.38, built on June 20, 2023

Are you getting hints for other properties of logbook, but not this one?

**Comment by piotrooo** [comment]
Yes, I have suggestions from the other projects. 
I using `application.properties`, maybe this is a reason?

**Comment by piotrooo** [comment]
Yes, I still have this issue. It's super weird... All properties (from 3rd party libs and from the project) are working. There is only problem with logbook :cry: 

A little bit info about my setup:
* Java
```
openjdk version "17.0.7" 2023-04-18
OpenJDK Runtime Environment Temurin-17.0.7+7 (build 17.0.7+7)
OpenJDK 64-Bit Server VM Temurin-17.0.7+7 (build 17.0.7+7, mixed mode, sharing)
```
* Spring Boot 3.1.2
* Logbook 3.2.0
* Ubuntu 22.04.2 LTS

The only difference that I have noticed in comparison with other libs is file in a `META-INF` `additional-spring-configuration-metadata.json` for logbook and `spring-configuration-metadata.json` for other libs.

![Screenshot from 2023-07-26 08-20-37](https://github.com/zalando/logbook/assets/2005054/bec24a8c-9fdc-40d1-b158-7d5b4773439a)

![Screenshot from 2023-07-26 08-21-08](https://github.com/zalando/logbook/assets/2005054/1c10db28-2952-4f25-aaf9-f7a965b1f9eb)

**Comment by kasmarian** [comment]
Both `additional-spring-configuration-metadata.json` and `spring-configuration-metadata.json` should have worked. Maybe something to do with the annotationProcessor as described in [3.1. Configuring the Annotation Processor](https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html#appendix.configuration-metadata.annotation-processor.configuring).
Alternatively, could you run a small experiment? Rename `additional-spring-configuration-metadata.json` to `spring-configuration-metadata.json`, run mvn install locally and try using the `3.4.0-SNAPSHOT` version. To verify if it's something to do with the metadata being defined in another type of the file

**Comment by star4j** [comment]
The same issue, I changed the file "additional-spring-configuration-metadata.json" to "spring-configuration-metadata.json," and the problem was resolved.
<img width="500" alt="image" src="https://github.com/zalando/logbook/assets/25194485/2da0f537-328c-4c20-ad14-e56838b0f0c1">

**Comment by kasmarian** [comment]
This should be resolved in the next release (3.5.0). Please re-open this issue if the problem is not fully addressed

**Comment by piotrooo** [comment]
I can confirm that fix solved the problem for me.
